src/search/documents/user.py

```diff
@@ -72,14 +73,24 @@ def prepare_author_profile(self, instance):
 
     # Used specifically for "autocomplete" style suggest feature
     def prepare_full_name_suggest(self, instance):
-        full_name_suggest = f"{instance.first_name} {instance.last_name}"
+        full_name_suggest = ""
+        try:
+            full_name_suggest = f"{instance.author_profile.first_name} {instance.author_profile.last_name}"
+        except Exception as e:
```

Comment:
Nit: Remove unused variable `e`.

```suggestion
        except Exception:
```